app/frontend/src/features/profile/actions/RespondToFriendRequestProfileButton.tsx

```diff
@@ -0,0 +1,116 @@
+import { CircularProgress } from "@material-ui/core";
+import Button from "components/Button";
+import IconButton from "components/IconButton";
+import { CheckIcon, CloseIcon, PersonAddIcon } from "components/Icons";
+import Menu, { MenuItem } from "components/Menu";
+import { PENDING } from "features/connections/constants";
+import { SetMutationError } from "features/connections/friends";
+import useRespondToFriendRequest from "features/connections/friends/useRespondToFriendRequest";
+import { FriendRequest } from "pb/api_pb";
+import React, { useRef, useState } from "react";
+
+interface RespondToFriendRequestProfileButtonProps {
+  friendRequestId: number;
+  state: FriendRequest.FriendRequestStatus;
+  setMutationError: SetMutationError;
+}
+
+export const RESPOND_TO_FRIEND_REQUEST_MENU_ID =
+  "respond-to-friend-request-actions-menu";
+
+function RespondToFriendRequestProfileButton({
```

Comment:
`RespondFriendRequestButton` is fine, since it's in the `profile` feature folder